src/NServiceBus.Core/Hosting/Helpers/AssemblyScanningComponent.cs

```diff
@@ -20,7 +22,10 @@ public static AssemblyScanningComponent Initialize(Configuration configuration,
             {
                 var directoryToScan = AppDomain.CurrentDomain.RelativeSearchPath ?? AppDomain.CurrentDomain.BaseDirectory;
 
-                assemblyScanner = new AssemblyScanner(directoryToScan);
+                assemblyScanner = string.IsNullOrWhiteSpace(assemblyScannerSettings.AdditionalAssemblyScanningPath)
```

Comment:
What was the reason for passing the additional path via constructor vs. adding another property like the other settings?

Reply:
Originally, we wanted to update the constructor to take `string[] params` but since it's a public type, we've decided to add an internal overload of the constructor.

Reply:
I'm asking why pass it through the constructor at all? There's already a pattern in place to pass in scanner settings.

Reply:
As a property, similar to [this](https://github.com/Particular/NServiceBus/blob/e8874d3e6c7a361f823ec47fda76ef3fc50b62f6/src/NServiceBus.Core/Hosting/Helpers/AssemblyScanningComponent.cs#L37-L41)? Yes, we could do that. Just not sure why the conventional scanning path was not done using the same pattern and instead was passed via the constructor. To keep "the pattern" in place, we've followed what was already done.

Reply:
Given that it is optional, I see how it makes sense. Will change.